DiscriminatedOnions.Tests/ReadMeExamples.cs

```diff
@@ -106,9 +106,8 @@ public static void Result_Bind_Error()
     public static async Task Result_BindAsync_Ok()
     {
         Result<string, int> ok = Result.Ok<string, int>("result value");
-        Result<string, int> asyncBoundOk = await ok
-            .BindAsync(v => Task.FromResult(Result.Ok<string, int>("beautiful " + v)))
-            .Pipe(o => o.BindAsync(v => Task.FromResult(Result.Ok<string, int>("very " + v))));
+        var bindAsyncResult = await ok.BindAsync(v => Task.FromResult(Result.Ok<string, int>("beautiful " + v)));
```

Comment:
I assume this one was just a test, because the original one, without the additional await, already suits the test case thanks to the other overload of Pipe.